asm/c128/stubs_test.go

```diff
@@ -0,0 +1,231 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package c128
+
+import "testing"
+
+var tests = []struct {
```

Comment:
Would it be better to also compare the assembly output to the full Go output on random values? This way it would be easier to write lots of tests. It would be nice to just create a bunch of tests where all of the combinations of unique and the same are tested
````
for _, incX := range []int{1, 2,3,-1, -2,-3}{
     for _, incY := range []int{1,2,3,-2,-3 }{ 
      ....
````